modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/util/v1_0/AttachmentUtil.java

```diff
@@ -194,7 +194,7 @@ public static CPAttachmentFileEntry addOrUpdateCPAttachmentFileEntry(
 			displayDateConfig.getMinute(), expirationDateConfig.getMonth(),
 			expirationDateConfig.getDay(), expirationDateConfig.getYear(),
 			expirationDateConfig.getHour(), expirationDateConfig.getMinute(),
-			GetterUtil.get(attachmentBase64.getNeverExpire(), false),
+			GetterUtil.get(attachmentBase64.getNeverExpire(), false), true,
```

Comment:
why true and not the GetterUtil?

Reply:
This is right, I overlooked this one and I'll change it immediately. Thanks!